roles/pulp_common/README.md

```diff
@@ -71,12 +71,14 @@ Role Variables
 * `pulp_user_home`: absolute path for pulp user home. Defaults to "/var/lib/pulp".
   This variable is also default for "{{ pulp_settings.deploy_root }}, which multiple directories
   are under. Also, `pulp_scripts_dir` is placed under it.
-* `pulp_certs_dir`: Path where to generate or drop the TLS certificates (see pulp_webserver role) &
-  keys for authentication tokens (see pulp_api role.) Also to where the user-provided gpg key for
-  the galaxy-ng collection signing service is placed (see galaxy_post_install role.) Defaults to
-  '{{ pulp_config_dir }}/certs' .
+* `pulp_certs_dir`: Path where to generate or drop the TLS certificates (see
+  [pulp_webserver](../../roles/pulp_webserver) role), the keys for authentication tokens (see
+  [pulp_api](../../roles/pulp_api) role), the database fields encryption key (See
+  [pulp_database_config](../../roles/pulp_database_config) role). Also to where the user-provided gpg
```

Comment:
have you checked if these links worked on mkdocs?

Reply:
Yes, they do.